### DISCUSSION TITLE
How to change scroll mode using JavaScript

### DISCUSSION THREAD

**Discussion by garthholmes** [CoderLine/alphaTab#1765] [Q&A] [answered]
Hi,

Looking at the following page:

https://alphatab.net/docs/reference/settings/player/scrollmode/

there is no example, as to how this may be used, so I have used the HTML attribute so if anyone could provide an example as to how this may be accomplished, it would be very much appreciated.

Very glad this library is still maintained and working well.

**Comment by Danielku15** [comment, marked as answer]
You might walk through the [tutorial](https://alphatab.net/docs/tutorial-web/introduction) to learn about these basics on settings and API inteactions. Some reference links:

* https://alphatab.net/docs/tutorial-web/introduction
* https://alphatab.net/docs/reference/settings (top section)
* https://alphatab.net/docs/reference/api/settings/
* https://alphatab.net/docs/reference/api/updatesettings/

Hope that helps finding out how to change settings.
